inference-engine/thirdparty/clDNN/src/include/program_node.h

```diff
@@ -305,7 +305,12 @@ struct program_node {
 
     bool need_lockable_memory() const;
 
+    std::string get_unique_id() const { return unique_id; }
+    void set_unique_id(std::string id) { unique_id = id; }
+
 protected:
+    std::string unique_id;
```

Comment:
Can we use `primitive_id org_id` in line 340 instead this variable?

Reply:
I think its not available. because unique_id is used to make KernelID in GetEntryPoint and the KernelID is consist of function name in CL kernel.

for example,
- org_id: greater:LSTMSequence_98/tensor_iterator/8/Greater_386_cldnn_in0_reshape
- kernelID is used to make kernel name and FUNC macro
// Kernel name: generic_eltwise_ref_1_0686
#define KERNEL(name) __kernel void generic_eltwise_ref_1_0686
#define FUNC(name)  _##name##_generic_eltwise_ref_1_0686